shell/platform/android/io/flutter/view/TextureRegistry.java

```diff
@@ -106,18 +106,65 @@ interface SurfaceProducer extends TextureEntry {
     /** Callback invoked by {@link #setCallback(Callback)}. */
     interface Callback {
       /**
-       * Invoked when a previous surface is now invalid and a new surface is now available.
+       * An alias for {@link Callback#onSurfaceAvailable()} with a less accurate name.
+       *
+       * @deprecated Override and use {@link Callback#onSurfaceAvailable()} instead.
+       */
+      @Deprecated(since = "Flutter 3.27", forRemoval = true)
+      default void onSurfaceCreated() {}
```

Comment:
default? 🤔

Reply:
Basically let folks who haven't used SurfaceProducer.Callback not have to define both methods.